<launch>
  <node pkg="iai_gripper_control"  type="iai_gripper_joint_state_republisher"  name="iai_gripper_joint_state_republisher" output="screen">
    <param name="joint_state_republish_topic_name" value="/joint_states" />
    <param name="joint_state_subscribe_topic_name" value="/vs087/joint_states" />
  </node>

</launch>